render/text.go

```diff
@@ -112,9 +127,9 @@ func (t *Text) Wrap(charLimit int, vertInc float64) []*Text {
 // of Modifications.
 func (t *Text) ToSprite() *Sprite {
 	width := t.d.MeasureString(t.text.String()).Round()
-	height := alg.RoundF64(t.d.Size)
-	s := NewEmptySprite(t.X(), t.Y()-float64(height), width, height)
-	t.DrawOffset(s.GetRGBA(), -t.X(), (-t.Y())+float64(height))
+	height := t.d.bounds.Max.Y
+	s := NewEmptySprite(t.X(), t.Y()-float64(height), width, height+5)
```

Comment:
height+5 seems magic, should this be default padding that can be manipulated?

Reply:
For future reference, this is a stopgap for #59